src/Select.js

```diff
@@ -556,10 +556,12 @@ class Select extends React.Component {
 	}
 
 	handleMenuScroll (event) {
-		if (!this.props.onMenuScrollToBottom) return;
```

Comment:
This is the only file I actually modified. Everything else was done by the build script.